src/encoded/static/components/experiment.js

```diff
@@ -232,13 +227,26 @@ const LibraryProperties = ({ replicates }) => {
                         </div>
                     );
                 } else {
+                    let strandSpecificity = null;
+
+                    if (isStrandSpecificity) {
+                        const strandSpecificValues = libraryPropertyElements.map(element => element.value);
+                        const strandSpecificityFirstValue = strandSpecificValues[0];
+
+                        if (strandSpecificValues.every(value => value === strandSpecificityFirstValue)) {
+                            strandSpecificity = `Strand-specific${strandSpecificityFirstValue === 'strand-specific' ? '' : ` (${strandSpecificityFirstValue})`}`;
+                        } else {
+                            strandSpecificity = 'Strand-specific (mixed)';
+                        }
+                    }
```

Comment:
Just to have this in the Github record, I think a future ticket should move this logic back to `libraryPropertyExtractors` which was designed for this kind of case, to avoid this special case.

Reply:
Good point. Still, I did not add the change to libraryPropertyExtractors because it looks at one strand_specificity at a time. To determine what to display, I needed to be able to examine all strands_specificities.

Reply:
The `replicates` object gets passed in the second parameter to the library extractors, so you have access to all libraries in the experiment. `size_range` uses this.